src/OrleansTestingHost/TestingSiloHost.cs

```diff
@@ -630,6 +694,11 @@ private static void StopOrleansSilo(SiloHandle instance, bool stopGracefully)
                 }
             }
 
+            // give silo a bit more time before unloading appdomain
+            // e.g. for completing async storage requests
+            if (stopGracefully)
```

Comment:
I think this is no longer needed with the latest implementation of the gossip protocol (it used to be much slower than it is now), tests are passing without it. I am removing it for now.